Sources/FluentKit/Migration/Migration.swift

```diff
@@ -1,6 +1,19 @@
+/// `Migration` can assist with database migration. It can choose
+/// to create a new database schema for migration, or drop an
```

Comment:
Language nit - we should tighten up the language here to avoid confusion. Migrations are database changes (such as adding new tables, changing existing tables or adding seed data) that are handled by Fluent and executed only once.